oviewer/draw.go

```diff
@@ -54,6 +55,7 @@ func (root *Root) drawBody(lX int, lN int) (int, int) {
 	markStyleWidth := min(root.scr.vWidth, root.Doc.general.MarkStyleWidth)
 
 	wrapNum := m.numOfWrap(lX, lN)
+	log.Println("wrapNum:", lN, wrapNum, m.width)
```

Comment:
Do you want to keep this?

It looks like a left behind debug

Reply:
Thank you for your comment.
However, this PR is experimental, so it may change a lot. Also, it may not be merged.